2023-03-31 10:43:38 database: flush
org.h2.message.DbException: Внутренняя ошибка: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/kovar/Desktop/servers/velocity/plugins/limboauth/limboauth-v2.mv.db [2.1.214/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/kovar/Desktop/servers/velocity/plugins/limboauth/limboauth-v2.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.elytrium.limboauth.dependencies.DatabaseLibrary.fromDriver(DatabaseLibrary.java:149)
	at net.elytrium.limboauth.dependencies.DatabaseLibrary.lambda$static$2(DatabaseLibrary.java:45)
	at net.elytrium.limboauth.dependencies.DatabaseLibrary.connect(DatabaseLibrary.java:108)
	at net.elytrium.limboauth.dependencies.DatabaseLibrary.connectToORM(DatabaseLibrary.java:124)
	at net.elytrium.limboauth.LimboAuth.reload(LimboAuth.java:297)
	at net.elytrium.limboauth.LimboAuth.onProxyInitialization(LimboAuth.java:207)
	at net.elytrium.limboauth.Lmbda$3.execute(Unknown Source)
	at com.velocitypowered.proxy.event.UntargetedEventHandler$VoidHandler.lambda$buildHandler$0(UntargetedEventHandler.java:56)
	at com.velocitypowered.proxy.event.VelocityEventManager.fire(VelocityEventManager.java:598)
	at com.velocitypowered.proxy.event.VelocityEventManager.lambda$fire$5(VelocityEventManager.java:479)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Внутренняя ошибка: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/kovar/Desktop/servers/velocity/plugins/limboauth/limboauth-v2.mv.db [2.1.214/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/kovar/Desktop/servers/velocity/plugins/limboauth/limboauth-v2.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 32 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/kovar/Desktop/servers/velocity/plugins/limboauth/limboauth-v2.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 26 more
2023-03-31 10:44:49 database: flush
org.h2.message.DbException: Внутренняя ошибка: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/kovar/Desktop/servers/velocity/plugins/limboauth/limboauth-v2.mv.db [2.1.214/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/kovar/Desktop/servers/velocity/plugins/limboauth/limboauth-v2.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.elytrium.limboauth.dependencies.DatabaseLibrary.fromDriver(DatabaseLibrary.java:149)
	at net.elytrium.limboauth.dependencies.DatabaseLibrary.lambda$static$2(DatabaseLibrary.java:45)
	at net.elytrium.limboauth.dependencies.DatabaseLibrary.connect(DatabaseLibrary.java:108)
	at net.elytrium.limboauth.dependencies.DatabaseLibrary.connectToORM(DatabaseLibrary.java:124)
	at net.elytrium.limboauth.LimboAuth.reload(LimboAuth.java:297)
	at net.elytrium.limboauth.LimboAuth.onProxyInitialization(LimboAuth.java:207)
	at net.elytrium.limboauth.Lmbda$3.execute(Unknown Source)
	at com.velocitypowered.proxy.event.UntargetedEventHandler$VoidHandler.lambda$buildHandler$0(UntargetedEventHandler.java:56)
	at com.velocitypowered.proxy.event.VelocityEventManager.fire(VelocityEventManager.java:598)
	at com.velocitypowered.proxy.event.VelocityEventManager.lambda$fire$5(VelocityEventManager.java:479)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Внутренняя ошибка: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/kovar/Desktop/servers/velocity/plugins/limboauth/limboauth-v2.mv.db [2.1.214/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/kovar/Desktop/servers/velocity/plugins/limboauth/limboauth-v2.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 32 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/kovar/Desktop/servers/velocity/plugins/limboauth/limboauth-v2.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 26 more
